src/main/java/com/epam/lowcost/service/implementations/FlightServiceImpl.java

```diff
@@ -44,4 +54,19 @@ public Flight deleteFlight(Long id) {
     public List<Flight> getByFromToDate(String departureAirport, String arrivalAirport, LocalDateTime departureDateFrom, LocalDateTime departureDateTo) {
         return flightDAO.getByFromToDate(departureAirport, arrivalAirport, departureDateFrom, departureDateTo);
     }
+
+    private long updateFlightPrice(Flight flight) {
```

Comment:
you update only initial price. Besides this you don't update flight, you only return new price

Reply:
Because i don't want to change flight initial price. I need initial price for recounting it dependending on the date.

Reply:
If you pass flight and want to update filght price - ok, return void, your price will be refreshed in method. But It bad practice. You business logic of new init price calculation based on initial price and range of date. Why function exists in method updateFlightPrice?

Reply:
I have some thinks how it can be implemented little bit differently. We will discuss about it tomorrow